.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
```

Comment:
Yes, this is repo-specific config. I copied it straight from the dependency-file-generator repo.

Reply:
I don't have a strong pref here although I would generally prefer to stick with either the black (88) or flake8 (79) defaults since I have found that other heuristics in linters are occasionally designed in such a way that they produce suboptimal results on very different line lengths. Perhaps that situation has improved though.